src/menu/MenuBar.ts

```diff
@@ -0,0 +1,96 @@
+import { createHandle } from '@dojo/core/lang';
+import { debounce } from '@dojo/core/util';
+import Observable from '@dojo/shim/Observable';
+import { v, w } from '@dojo/widget-core/d';
+import ThemeableMixin, { theme, ThemeableProperties } from '@dojo/widget-core/mixins/Themeable';
+import WidgetBase from '@dojo/widget-core/WidgetBase';
+import * as css from './styles/menuBar.m.css';
+import SlidePane from '../slidepane/SlidePane';
+
+export interface MenuBarProperties extends ThemeableProperties {
+	breakpoint: number;
+	slidePaneButtonLabel?: string;
+	slidePaneStyles?: any;
+	onRequestClose?: () => void;
+	onRequestOpen?: () => void;
+	open?: boolean;
+}
+
+// TODO: Make this a hot observable with Evented?
+// TODO: Should this be moved into a common utility method that can be used by any widget?
```

Comment:
Could you remove these? Also, I do think that this should go into a utility method in `common`, this is nice and useful.